lib/util/hud_lists.rb

```diff
@@ -47,16 +47,16 @@ def record_type_map
     {
       12 => 'Contact 12',
       13 => 'Contact 13',
-      141 => 'PATH service',
-      142 => 'RHY service connections',
-      143 => 'HOPWA service',
-      144 => 'SSVF service',
-      151 => 'HOPWA financial assistance',
-      152 => 'SSVF financial assistance',
-      161 => 'PATH referral',
-      162 => 'RHY referral',
-      200 => 'Bed night',
-      210 => 'HUD-VASH OTH voucher tracking',
+      141 => 'PATH Service',
+      142 => 'RHY Service Connections',
+      143 => 'HOPWA Service',
+      144 => 'SSVF Service',
+      151 => 'HOPWA Financial Assistance',
+      152 => 'SSVF Financial Assistance',
+      161 => 'PATH Referral',
+      162 => 'RHY Referral',
+      200 => 'Bed Night',
+      210 => 'HUD-VASH OTH Voucher Tracking',
```

Comment:
Hmm, my only hesitation its that the HMIS CSV spec lists them as they were.  I'm not sure it'll ever matter, since we should always be getting them as numbers, but I thought I'd mention it.
<img width="1011" alt="Screenshot 2023-02-13 at 4 55 47 PM" src="https://user-images.githubusercontent.com/1346876/218583523-f85ecfde-e87b-47cf-94f7-1b991dbddf63.png">

Reply:
Yeah, we definitely shouldn't be mapping by description anywhere. If you prefer these to remain as-is, I can move the cast changes into the enum generation code.

Reply:
Let's make this change and keep an eye on it.  The only places I can think we might be trying to match an exact string is in some incoming non-HMIS data.

Reply:
Oh, and maybe the failing test?